### PR TITLE
doc: added documentation of CLI Oauth

### PR DESCRIPTION
**Pull request by warber**
closes https://github.com/keptn/keptn/issues/6086

### CODE REVIEW

File: content/docs/0.13.x/operate/user_management/openid_authentication/index.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ description: How to setup user authentication using OIDC
 weight: 10
 ---
 
-To set up user authenticating using OIDC, customization of your **Keptn Bridge** is needed. 
+Please follow the following links for instructions how to setup user authentication using OIDC:
 
-* Please follow the instructions provided [here](../../../reference/bridge/oauth/#enable-disable-authentication).
+* [Keptn Bridge](../../../reference/bridge/oauth/#enable-disable-authentication)
+* [Keptn CLI](../../../reference/cli/commands/keptn_auth)
```
Comment (odubajDT) [inline content/docs/0.13.x/operate/user_management/openid_authentication/index.md:10]: This link does not work

File: content/docs/0.13.x/operate/user_management/microsoft/index.md [hunk 2]
```diff
@@ -33,15 +35,28 @@ OAUTH_CLIENT_SECRET: <client_secret>
...
 
-* After the user successfully logs in with his Microsoft credentials, the user is redirected back to the Bridge. Once redirected, the Bridge server fetches the user tokens and creates a session. The user is now successfully logged in.
+After the user successfully logs in with his Microsoft credentials, the user is redirected back to the Bridge. Once redirected, the Bridge server fetches the user tokens and creates a session. The user is now successfully logged in.
 {{< popup_image
 link="./assets/oauth-logged-in.png"
 caption="User is logged in" width="800px">}}
+
+## Keptn CLI
+
+The `keptn auth` command provides several [command line flags](../../../reference/cli/commands/keptn_auth) which can be used to setup the CLI to use SSO via OpenID:
```
Comment (odubajDT) [inline content/docs/0.13.x/operate/user_management/microsoft/index.md:53]: this link does not work

File: content/docs/0.13.x/operate/user_management/openid_authentication/index.md (hunk 1 above)
Comment (warber) [reply content/docs/0.13.x/operate/user_management/openid_authentication/index.md:10]: in the netlify preview they seem to work

File: content/docs/0.13.x/operate/user_management/microsoft/index.md (hunk 2 above)
Comment (warber) [reply content/docs/0.13.x/operate/user_management/microsoft/index.md:53]: same as above, why do you think it's not working?

File: content/docs/0.13.x/operate/user_management/openid_authentication/index.md (hunk 1 above)
Comment (odubajDT) [reply content/docs/0.13.x/operate/user_management/openid_authentication/index.md:10]: it just gives me github 404 site when I click on it here in github, but when adding ".md" to the end, it works

Comment (odubajDT) [reply content/docs/0.13.x/operate/user_management/openid_authentication/index.md:10]: try here https://github.com/keptn/keptn.github.io/blob/e5ccbe837b4f3fe2e06cf7f1f813944c5fe546c7/content/docs/0.13.x/operate/user_management/openid_authentication/index.md

Comment (warber) [reply content/docs/0.13.x/operate/user_management/openid_authentication/index.md:10]: you need to try here: https://deploy-preview-1072--keptn.netlify.app/docs/0.13.x/operate/user_management/openid_authentication/

Comment (thisthat) [reply content/docs/0.13.x/operate/user_management/openid_authentication/index.md:10]: These Markdown files are compiled into a website using [Hugo](https://gohugo.io/). That's why your link doesn't work @odubajDT  :)

File: content/docs/0.13.x/operate/user_management/microsoft/index.md (hunk 2 above)
Comment (thisthat) [inline content/docs/0.13.x/operate/user_management/microsoft/index.md:53]:
```suggestion
The `keptn auth` command provides several [command line flags](../../../reference/cli/commands/keptn_auth) which can be used to setup the CLI to use an SSO via OpenID Connect:
```

File: content/docs/0.13.x/operate/user_management/microsoft/index.md [hunk 3]
```diff
@@ -33,15 +35,28 @@ OAUTH_CLIENT_SECRET: <client_secret>
...
+
+```
+keptn auth --oauth --oauth-discovery https://login.microsoftonline.com/<directory_tenant_id>/v2.0/.well-known/openid-configuration --oauth-client-id <client_id> --oauth-client-secret <client_secret> --endpoint <keptn_endpoint> --api-token <keptn_api_token>
+```
+
+After executing the `keptn auth` command, a Browser window should open asking you to confirm the login using your credentials.
+If everything went well you are redirected to an HTML page that confirms that the login was successful.
```
Comment (thisthat) [inline content/docs/0.13.x/operate/user_management/microsoft/index.md:60]:
```suggestion
If everything went well, you are redirected to an HTML page that confirms that the login was successful.
```

File: content/docs/0.13.x/operate/user_management/openid_authentication/index.md [hunk 4]
```diff
@@ -4,10 +4,11 @@ description: How to setup user authentication using OIDC
 weight: 10
 ---
 
-To set up user authenticating using OIDC, customization of your **Keptn Bridge** is needed. 
+Please follow the following links for instructions how to setup user authentication using OIDC:
```
Comment (thisthat) [inline content/docs/0.13.x/operate/user_management/openid_authentication/index.md:7]:
```suggestion
Please, follow these links for instructions on how to set up user authentication using OIDC:
```

**Review by thisthat — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
Looks good. Not sure whether one expects this documentation. It is for OIDC, hence we need it only for one baseline

**Pull request merged**